server/konnectors/googlecontacts.coffee

```diff
@@ -0,0 +1,342 @@
+request = require 'request'
+async = require 'async'
+fetcher = require '../lib/fetcher'
+extend = require('util')._extend
+
+Contact = require '../models/contact'
+Tag = require '../models/tag'
+
+CompareContacts = require '../lib/compare_contacts'
+ContactHelper = require '../lib/contact_helper'
+GoogleContactHelper = require '../lib/google_contact_helper'
+
+GoogleToken = require '../lib/google_access_token'
+
+
+localization = require '../lib/localization_manager'
+
+log = require('printit')
+    prefix: "Google Contacts"
+    date: true
+
+
+ACCOUNT_TYPE = 'com.google'
+
+# Konnector
+
+module.exports =
+
+    name: "Google Contacts"
+    slug: "googlecontacts"
+    description: "Synchronize google contacts with cozy through google's API. Experimental - please backup your contacts from your cozy and your google account."
+    vendorLink: "https://www.google.com/contacts/"
+
+    customView: """
+    <h6>Initialize or reset this konnector</h6>
+    <p>1. Press "connect your google account" button to connect to your Google account and authorize your Cozy to access to it. Google will provide you with a complex string. Once you get it copy it in your clipboard, we will use it in second step.</p>
+    <button id="connect-google" title="Connect your Google account" class="btn"
+       onclick="window.open('#{GoogleToken.getAuthUrl()}', 'Google OAuth', 'toolbars=0,width=700,height=600,left=200,top=200,scrollbars=1,resizable=1'); var input = $('#googlecontacts-authCode-input'); input.parents('.field').toggleClass('hidden'); input.attr('type', 'text'); input.val(''); $('#googlecontacts-accountName-input').val('');return false;"
```

Comment:
Should we keep this hack ? Some js is mandatory (the window.open) , but most is just here for UX.
If yes, we will need a solution for translations.